Dotnet/AppApi/GameHandler.cs

```diff
@@ -54,6 +54,19 @@ public int QuitGame()
             return processes.Length;
         }
 
+        /// <summary>
+        /// Kills the install.exe process after exiting game.
+        /// </summary>
+        /// <returns>The number of processes that were killed (0 or 1).</returns>
+        public int KillInstall()
+        {
+            var processes = Process.GetProcessesByName("install");
```

Comment:
I'm a bit hesitant about the way `install.exe` is being grabbed here.  While it makes sense for `vrchat` because the likelihood of the same application existing is little to none, `install` can very much encompass more processes that may be unintentionally killed.

Reply:
Detect if install.exe is in VRChat folder with regular expression